common/filestore_utils.py

```diff
@@ -27,16 +31,35 @@ def _using_gsutil():
     return experiment_filestore_path.startswith('gs://')
 
 
-if _using_gsutil():
-    from common import gsutil as filestore_utils_impl
-else:
-    # When gsutil is not used in the context, here it should use local_utils.
-    # TODO(zhichengcai): Implement local_filestore.py and import it here.
-    from common import gsutil as filestore_utils_impl
+global filestore_utils_impl
```

Comment:
is this needed? Why not just do it on line 48?

Reply:
Let me check this later.